contracts/v0.6/LinkToken.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import './token/LinkERC20.sol';
+import './ERC677Token.sol';
+
+contract LinkToken is ERC20, LinkERC20, ERC677Token {
+  uint private constant TOTAL_SUPPLY = 10**27;
+  string private constant NAME = 'ChainLink Token';
+  string private constant SYMBOL = 'LINK';
+
+  constructor() ERC20(NAME, SYMBOL)
```

Comment:
```suggestion
  constructor() LinkERC20(NAME, SYMBOL)
```

Reply:
This will not work as  `LinkERC20` does not have a constructor itself.

Reply:
Ok, can you still call the ERC20 constructor without inheriting from it directly?

Reply:
Yes!